service-workers/service-worker/navigation-preload/get-state.https.html

```diff
@@ -14,204 +14,159 @@
-                                        'state after disable() (from worker)');
-        expect_navigation_preload_state(states[1], false, 'true',
-                                        'state after disable() (from page)');
-        return post_and_wait_for_reply(worker, 'setHeaderValue');
-      })
-    .then(data => {
-        assert_equals(data, undefined,
-                      '.setHeaderValue() should resolve to undefined');
-        return Promise.all([
+  const registration =
+      await service_worker_unregister_and_register(t, SCRIPT, SCOPE);
+
+  add_completion_callback(async() => {
+    if (registration)
+      await registration.unregister();
+  })
+
+  const worker = registration.installing;
+  await wait_for_state(t, worker, 'activated');
+  let data = await post_and_wait_for_reply(worker, 'getState');
+  let states = await Promise.all([data,
+                                 registration.navigationPreload.getState()]);
+  expect_navigation_preload_state(states[0], false, 'true',
+                                  'default state (from worker)');
+  expect_navigation_preload_state(states[1], false, 'true',
+                                  'default state (from page)');
+
+  data = await post_and_wait_for_reply(worker, 'enable');
+  assert_equals(data, undefined, 'enable() should resolve to undefined');
+  states = await Promise.all([
             post_and_wait_for_reply(worker, 'getState'),
             registration.navigationPreload.getState()]);
-      })
-    .then(states => {
-        expect_navigation_preload_state(
-            states[0], false, 'insightful',
-            'state after setHeaderValue() (from worker)');
-        expect_navigation_preload_state(
-            states[1], false, 'insightful',
-            'state after setHeaderValue() (from page)');
-      });
-  }, 'getState from a worker');
+  expect_navigation_preload_state(states[0], true, 'true',
+                                  'state after enable() (from worker)');
+  expect_navigation_preload_state(states[1], true, 'true',
+                                  'state after enable() (from page)');
+
+  // Call disable() and then getState().
+  data = await post_and_wait_for_reply(worker, 'disable');
+  assert_equals(data, undefined,
+                '.disable() should resolve to undefined');
+  states = await Promise.all([
+      post_and_wait_for_reply(worker, 'getState'),
+      registration.navigationPreload.getState()]);
+  expect_navigation_preload_state(states[0], false, 'true',
+                                  'state after disable() (from worker)');
+  expect_navigation_preload_state(states[1], false, 'true',
+                                  'state after disable() (from page)');
+
+  data = await post_and_wait_for_reply(worker, 'setHeaderValue');
+  states = await Promise.all([
+           post_and_wait_for_reply(worker, 'getState'),
+           registration.navigationPreload.getState()]);
```

Comment:
This formatting is inconsistent with 121-123. To save time, I'd just run this through prettier.io.